Buildings/Templates/ChilledWaterPlant/Components/Chiller/Interfaces/Chiller.mo

```diff
@@ -0,0 +1,41 @@
+within Buildings.Templates.ChilledWaterPlant.Components.Chiller.Interfaces;
+partial model Chiller
```

Comment:
OK to keep the Chiller classes because

- the type compatibility between the different classes within `Buildings.Fluid.Chillers` is not guaranteed (whereas it is guarenteed by design for the classes within `Buildings.Templates.ChilledWaterPlant.Components.Chiller`),
- looking ahead, we may need to implement a wrapper (or container) class for the chillers to allow having different chiller types within the same array `chi[nChi]`.

What about the performance records? Did you try to construct an array of different record types (for instance from `Buildings.Fluid.Chillers.Data.ElectricEIR`) and pass that array as a parameter binding to `chi[nChi]`?
Or are we limited to use the same performance record for all elements of `chi[nChi]`? (Which would be a strong limitation.)